babel-macro/package.json

```diff
@@ -7,7 +7,7 @@
   ],
   "description": "",
   "scripts": {
-    "build": "rm -rf dist && tsc && cp -r src/test/backend/ dist/test/",
+    "build": "rm -rf dist && tsc && cp -r src/test/backend dist/test/",
```

Comment:
why did you make this change? it can cause a `dist/test/backend/backend` to be created